api/src/main/java/com/redhat/ipaas/api/v1/model/ConnectorGroup.java

```diff
@@ -21,30 +21,30 @@
 import java.io.Serializable;
 
 /**
- * ComponentGroups are labels in Camel.
+ * ConnectorGroups are labels in Camel.
```

Comment:
I probably have to dive deeper into the code, but is it correct when I assume that when the UI requests the `Connectors` for a  `ConnectorGroup` then ipaas-rest does a live-query on the `Connectors` which carry a certain label ? So a `ConnectorGroup` persistent attributes are the name of the group + the label for selecting the Camel connectors ?

Or in other words: A `ConnectorGroup` does *not* have a direct (db) reference to the `Connector` objects it contains in the DB (which needs to be maintained) ?

If so, I'm somewhat fine with calling it `ConnectorGroup` but handling it like a label.